src/ipc/ipc4/handler.c

```diff
@@ -1674,9 +1682,14 @@ void ipc_cmd(struct ipc_cmd_hdr *_hdr)
 
 		msg_reply.header = reply.primary.dat;
 
+#ifdef CONFIG_DEBUG_IPC_TIMINGS
+		tr_info(&ipc_tr, "tx-reply\t: %#x|%#x to %#x|%#x in %llu us", msg_reply.header,
+			msg_reply.extension, req.primary.dat, req.extension.dat,
+			k_cyc_to_us_near64(sof_cycle_get_64() - tstamp));
```

Comment:
this function only prepares IPCs and `ipc4_send_reply()` below only queues them, they're actually sent in `ipc_send_queued_msg()` in the IPC work handler. But maybe indeed for our purposes we only need the processing time, not the request-response time, just something to be aware of. So maybe the Kconfig description isn't quite correct - it's not before sending, it's before queuing

Reply:
Should this not be E2E timing, i.e we stop the timer when we actually send the reply ?

Reply:
> Should this not be E2E timing, i.e we stop the timer when we actually send the reply ?

@lgirdwood I guess both are needed for different kinds of analysis - depending on what we want to measure.